CHANGELOG.md

```diff
@@ -59,6 +67,8 @@ Release built: _not released yet_
         - Renamed `entity_vault_history` to `vault_balance_history`. Holds information about vault content (amount of fungibles or count of non fungible ids inside vault) at a given state version.
     - Key value store
         - New `key_value_store_totals_history` table, which holds total count of all keys under a given store at a given state version.
+- Changed `receipt_state_updates` in the `ledger_transactions` table to be nullable.
+- Moved all `receipt_event_*` columns from `ledger_transactions` to separate `ledger_transaction_events` table.
```

Comment:
Out of interest, why did we move the events to a separate table, but not also the state updates?

Reply:
because all events_ columns related to events and they are logically separate concept to rest of receipt. I don't think it makes sense to create separate table for just one column. i.e we could in future create one for all "raw" receipt data that's basically forwarded from core API.